### PR TITLE
Fix openshift4 tests

### PR DESCRIPTION
**Pull request by phracek**
This pull request fixes OpenShift 4 tests

### CODE REVIEW

**Comment by phracek** [comment]
This pull request fixes only OpenShift 4 tests. Container tests are not needed to run.

[test-openshift]

File: test/test-lib-mysql.sh [hunk 1]
```diff
@@ -114,9 +114,9 @@ function test_mysql_s2i() {
 
 function test_mariadb_integration() {
   local service_name=mariadb
-  ct_os_template_exists mariadb-ephemeral && t=mariadb-ephemeral || t=mariadb-persistent
+  ct_os_template_exists mariadb-ephemeral && t=mariadb-ephemeral-template.json || t=mariadb-persistent-template.json
```
Comment (phracek) [inline test/test-lib-mysql.sh:117]: Does it make sense to split tests are run as ephemeral as persistent?

Comment (zmiklank) [reply test/test-lib-mysql.sh:117]: If I understand this correctly; if we decided to use local template `mariadb-ephemeral-template.json` we do not need to check with `ct_os_template_exists` if it already exist in the openshift do we? Then I would suppose using testing both templates.

However, haven't this test aimed at testing a template, which is already available in the openshift?

Comment (phracek) [reply test/test-lib-mysql.sh]: Let's test both of them.

**Comment by phracek** [comment]
[test-all]

**Comment by hhorak** [comment]
LGTM.

File: test/test-lib-mysql.sh (hunk 1 above)
Comment (zmiklank) [reply test/test-lib-mysql.sh]: And could you please explain to me, what is the difference between using the local template and the template available in the OCP?

Comment (zmiklank) [reply test/test-lib-mysql.sh]: Thank you, I think I understand. In that case why do we not pull the templates from the repository but use the local ones?

Comment (pkubatrh) [reply test/test-lib-mysql.sh]: I expect we use local ones so we can test out changes to templates maintained in this repository as part of the PR.

Comment (zmiklank) [reply test/test-lib-mysql.sh]: Makes sense, thanks.

**Pull request merged**